digital-deck-ui/src/components/CardDeck.vue

```diff
@@ -1,48 +1,61 @@
 <template>
-    <div v-if="$store.getters.getNumCardsInDeck > 0"
-         class="card-deck"
-         :style="'background-image: url(\'' + deckImage + '\')'"
-         @click="drawCard" />
+  <div
+    v-if="$store.getters.getNumCardsInDeck > 0"
+    class="card-deck"
+    :style="
+      'background-image: url(\'' +
+      deckImage +
+      '\'); width: 144px; transform: translate(' +
```

Comment:
You can add `width: 144px` in the css class below since it's not reactive.

Reply:
In fact, you don't even need to do this whole `width` and `transform` thing. You can achieve the same by changing the `grid-template-columns` in `HandArea.vue` to be something like `9fr 1fr`

Reply:
I can do it in styles, but as for second point. I did this because then the click area for some reason was bigger than than card image itself which it shouldn't be.

Reply:
Which click area do you mean? If you mean the card deck it's pretty precise on my end

Reply:
Yes, deck. Do you still have the side area? Because if you remove that then I had troubles.

Reply:
Which side area? You mean the one on the left?

Reply:
Yes, I believe you have some placeholder text.